esx_service/vmdk-opsd

```diff
@@ -30,6 +28,14 @@ MAX_QUICK_FAILURES=5
 SIGTERM=15
 SIGKILL=9
 
+# Config
+VMDK_OPSD_PORT=1019 # Override using CONFIG_FILE
+
+# Create the following file if defaults need to be overridden
+# Example:
+# export VMDK_OPSD_PORT=1020
+CONFIG_FILE=/etc/vmdk_opsd/config.sh
```

Comment:
our log config is in `/etc/vmware/vmdkops/log_config.json`
I think  the config.sh should be in the same place, and desirably named using the same convention (e.g. service_config.sh to differentiate from log config) 
Or log_config could move